tests/ha/crm_mon.pm

```diff
@@ -8,18 +8,20 @@
 # without any warranty.
 
 # Summary: Check cluster status in crm_mon
-# Maintainer: Denis Zyuzin <dzyuzin@suse.com>
+# Based on script by Denis Zyuzin
+# Maintainer: Loic Devulder <ldevulder@suse.com>
 
-use base "hacluster";
+use base 'hacluster';
 use strict;
 use testapi;
 use autotest;
 use lockapi;
 
 sub run {
-    type_string "crm_mon -1\n";
+    # Show cluster informations
+    script_run q(crm_mon -R -1);
```

Comment:
I should have change type_string by assert_string_run the first time!
So yes, I will change script_run by assert_script_run in the next commit.